### DISCUSSION TITLE
使用域名访问权限登录不通过

### DISCUSSION THREAD

**Discussion by qingfengJava** [hhyo/Archery#2244] [General]
项目使用docker-compose已经部署完成，并且使用IP访问没有问题，但是配置了一个域名，使用nginx去路由，此时登录就受限制。目前不太清楚怎么去解决这个权限问题，麻烦帮助一下。
![image](https://github.com/hhyo/Archery/assets/97382270/2391e115-b919-498a-a733-b8f0f877b8f8)

**Comment by LeoQuote** [comment]
django4.0开始新增了origin host检查，请注意参数配置，否则会出现403错误，参考https://docs.djangoproject.com/en/4.0/ref/settings/#csrf-trusted-origins

**Reply by qingfengJava** [reply]
origin host检查的问题我解决了，也成功登录了，现在配置的数据库在查询上还有点问题，域名和数据库的配置这些都是用的服务器内网，它应该是没连接到数据库，这个我应该是在哪里进行配置还是怎么操作，我有点懵了，麻烦解决一下
![image](https://github.com/hhyo/Archery/assets/97382270/437a02d1-804a-4135-9551-1db4ceee2611)

**Reply by qingfengJava** [reply]
![image](https://github.com/hhyo/Archery/assets/97382270/b610e68e-c795-42b0-a047-8ee590c7dc16)
